extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.bindings.js

```diff
@@ -0,0 +1,87 @@
+angular.module('PaperUI.controllers.configuration.bindings', [ 'ngRoute' ]).config([ '$routeProvider', function($routeProvider) {
+    // Configure the routes for this controller
+    $routeProvider.when('/configuration/bindings', {
```

Comment:
Thats a good argument, lets see how it feels when we start changing the other controller implementations too. thanks for the discussion!